src/bundle/Resources/views/content/tabs/translations.html.twig

```diff
@@ -15,10 +19,12 @@
     <tbody>
         {% for translation in translations %}
         <tr>
-            <td></td>
+            <td>{{ form_widget(actionsForm.removeTranslations[translation.languageCode], {'attr': {'disabled': not translation.canDelete}}) }}</td>
```

Comment:
It looks like this is only checked on the front end. Do we not need some server side validation for this?